src/common/schemas/input/compute-ledger-hash.json

```diff
@@ -5,7 +5,8 @@
   "properties": {
     "ledger": {
       "$ref": "getLedger",
-      "description": "The ledger header to hash."
+      "description": "The ledger header to hash.",
+      "position": 0
```

Comment:
Oh, that makes sense. But I think you can still partially use it because I think there is only ever one optional argument per method. You could use jayson's technique for the required arguments, then get all the parameters from the schema and if there is just one extra, that is the last argument, and if there is more than one extra, just throw an exception.